Dockerfile

```diff
@@ -1 +1,52 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+ENV PENTAHO_CE_VERSION=8.1.0.0-365 && \
+    MYSQL_CONNECTOR_VERSION=8.0.24 && \
+    MARIADB_CONNECTOR_VERSION=2.2.6 && \
+    ARKCASE_PRE_AUTH_VERSION=4-1.1.1
+ARG RESOURCE_PATH="artifacts"
 
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_pentaho \
+      MAINTAINER="Armedia LLC"
+
+#-----------PENTAHO SETUP-------------------------------------
+RUN mkdir -p /home/pentaho/data/pentaho &&\
```

Comment:
Please use an ENV  or ARG depending on use case (build only use ARG, build + runtime use ENV) to specify the base path (/home/pentaho) vs hard coding it.  In this case, if we need to change it, we only need to do it in 1 place.

Reply:
@ymuwakki  Added base path to ENV